BUILD.bazel

```diff
@@ -665,6 +665,7 @@ cc_library(
         ":val_test_lib",
         "@googletest//:gtest",
         "@googletest//:gtest_main",
+        "@re2//:re2",
```

Comment:
let's not do this.  Let's weaken the regexp instead.

Reply:
I wasn't sure whether you'd mind the dependency in tests only. It'd be nice to have portable regexp syntax for tests. Test writers are currently limited to what happens to work across the three different syntaxes that Googletest uses depending on the platform. I'm fine with weakening the regexps though.